Purpose: Disable the SUSE flag.
Fixes:   None, made as part of Debian porting.
Authour: Kel Modderman
Status:  Debian specific, showed to upstream.
---
--- a/Makefile
+++ b/Makefile
@@ -1,5 +1,5 @@
 VERSION = 0.53.1
-ISSUSE	= -DSUSE
+ISSUSE	= -DNOTSUSE
 
 INSTALL		= install -m 755
 INSTALL_DATA	= install -m 644
